### PR TITLE
perf(draw): Updates forEach to for for speedy looping

### PR DESCRIPTION
**Pull request by 2xAA**
Changes `forEach` to `for` in main critical loop.
RE: #293

### CODE REVIEW

File: src/modv/draw.js [hunk 1]
```diff
@@ -90,14 +93,26 @@ function draw(δ) {
         }
       }
 
-      if (!enabled || alpha === 0) return
+      if (!enabled || alpha === 0) {
```
Comment (TimPietrusky) [inline src/modv/draw.js:96]: Why did this change from `please don't go on` to `let's continue`?

File: src/modv/draw.js [hunk 2]
```diff
@@ -90,14 +93,26 @@ function draw(δ) {
...
+      for (
+        let moduleIndex = 0;
+        moduleIndex < moduleOrderLength;
+        ++moduleIndex
+      ) {
+        const moduleName = Layer.moduleOrder[moduleIndex]
         const Module = store.getters['modVModules/outerActive'][moduleName]
 
-        if (!Module) return
+        if (!Module) {
```
Comment (TimPietrusky) [inline src/modv/draw.js:109]: Why did this change from `please don't go on` to `let's continue`?

File: src/modv/draw.js [hunk 3]
```diff
@@ -90,14 +93,26 @@ function draw(δ) {
...
+          continue
+        }
 
-        if (!Module.meta.enabled || Module.meta.alpha === 0) return
+        if (!Module.meta.enabled || Module.meta.alpha === 0) {
```
Comment (TimPietrusky) [inline src/modv/draw.js:113]: Why did this change from `please don't go on` to `let's continue`?

**Review by TimPietrusky — CHANGES_REQUESTED** [review]
@2xAA I have 3 tiny questions!

File: src/modv/draw.js (hunk 1 above)
Comment (2xAA) [reply src/modv/draw.js:96]: In `forEach` loop a `return` acts similarly to a `continue` in a `for` loop.

If I had left the `return` in there, any Module with no opacity, that wasn't enabled or any layer with no modules would have stopped the rest of the draw loop entirely.

File: src/modv/draw.js (hunk 2 above)
Comment (2xAA) [reply src/modv/draw.js:109]: In `forEach` loop a `return` acts similarly to a `continue` in a `for` loop.

If I had left the `return` in there, any Module with no opacity, that wasn't enabled or any layer with no modules would have stopped the rest of the draw loop entirely.

File: src/modv/draw.js (hunk 3 above)
Comment (2xAA) [reply src/modv/draw.js:113]: In `forEach` loop a `return` acts similarly to a `continue` in a `for` loop.

If I had left the `return` in there, any Module with no opacity, that wasn't enabled or any layer with no modules would have stopped the rest of the draw loop entirely.

**Review by 2xAA — COMMENTED** [review]
In a `forEach` loop a `return` acts similarly to a `continue` in a `for` loop.

If I had left the `return` in there, any Module with no opacity, that wasn't enabled or any layer with no modules would have stopped the rest of the draw loop entirely.

**Review by TimPietrusky — APPROVED** [review]

**Pull request merged**